django_app/settings.py

```diff
@@ -39,6 +45,7 @@
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
+    # 'whitenoise.runserver_nonstatic',
```

Comment:
Whitenoise configuration does not involve adding it to app list, kindly remove totally instead of commenting it out.